pkg/cli/admin/release/info.go

```diff
@@ -249,7 +251,14 @@ func replaceStableSemanticArgs(args []string, semanticArgs map[string]semver.Ver
 
 		var found bool
 		for _, stream := range []string{"fast", "stable", "candidate"} {
-			u.RawQuery = url.Values{"channel": []string{fmt.Sprintf("%s-%d.%d", stream, v.Major, v.Minor)}}.Encode()
+			params := url.Values{"channel": []string{fmt.Sprintf("%s-%d.%d", stream, v.Major, v.Minor)}}
+			//the api only accpets the arch portion of os/arch privided by filter-by-os parameters, so we filter out linux/ prefix
+			if filterByOs.FilterByOS != "" {
+				arch := filterByOs.FilterByOS
+				arch = strings.ReplaceAll(arch, "linux/", "")
+				params.Add("arch", arch)
+			}
+			u.RawQuery = params.Encode()
```

Comment:
@Prashanth684 @atiratree 
@wking 's comment about the overlap between `filter-by-os` and `arch` made me rethink this. After release payloads are manifest listed by default, we'll still want to get old releases by arch from the api. We probably don't want users to have to know which option to use.

For this to work, I think we'll need a plan for what will be returned from these endpoints:

`https://api.openshift.com/api/upgrades_info/v1/graph`
and 
`https://api.openshift.com/api/upgrades_info/v1/graph?arch=<arch> `

I assume that older releases (like 4.10) that aren't manifest listed will still be available? In other words, we aren't going to retroactively MF older releases?

Also, does it make sense to add a `filterbyos` param on the api? Feels wrong to filter `linux` here.

Reply:
i'm wondering how `--filter-by-os` will work for multi arch payloads. if the graph name is "multi", it doesn't make sense to pass in filter-by-os i guess. the arch option would probably be better in that case?

Reply:
Prashanth and I talked offline, reverting back to using `--arch` to avoid confusion between single arch payloads and `multi` release payloads.